browser.js

```diff
@@ -146,13 +146,14 @@ function wrap (opts, logger, val, level) {
     return function LOG () {
       var ts = Date.now()
       var args = new Array(arguments.length)
+      var proto = (Object.getPrototypeOf && Object.getPrototypeOf(this) === _console) ? _console : this
```

Comment:
As per your suggestion @davidmarkclements. Let me know if this is what you had in mind :)